vignettes/articles/hubEnsembles.Rmd

```diff
@@ -281,19 +291,55 @@ weighted_linear_pool_norm <- hubExamples::forecast_outputs |>
+When specifically requesting a linear pool made up of a **subset of the input sample predictions**, the user must identify the dependence structure using the `compound_taskid_set` parameter to ensure the resulting ensemble is valid. The compound task ID set consists of task ID variables that, together, identify a "compound modeling task" corresponding to a single modeled unit with a multivariate outcome of interest. Samples are expected to capture dependence across different outcomes within the same compound task (not across compound tasks), as each compound task is assumed to be independent of all other compound tasks. 
+
+For example, a compound task could be predicting the number of weekly incident influenza hospitalizations (`"target"`) in Massachusetts (`"location"`) starting on November 19, 2022 (`"reference_date"`). Here, `"horizon"` is not part of the compound task ID set, indicating that sample predictions made at each horizon depend on those for the previous horizon (i.e., 2 week ahead forecasts depend on 1 week ahead forecasts) within every compound task for the sample output type. Conversely, dependence across any compound tasks (such as a relationship between targets like influenza hospitalizations and deaths) would not be captured based on the current specification of the compound task ID set.
+
+If samples are not expected to capture dependence across the levels of any task ID variables (that is, we are summarizing predictions for marginal distributions), the `compound_taskid_set` consists of all task ID variables. Please see the [hubverse documentation on samples](https://hubverse.io/en/latest/user-guide/sample-output-type.html) for more information about compound tasks.
+
+Thus, in this example, those three task id variables (`"reference_date"`, `"location"`, and `"target"`) make up the compound task ID set that is specified in the call to `linear_pool()`. The `"target_end_date"` for a given forecast is derived from the combination of `"reference_date"` and `"horizon"`, and so it is specified as the argument for `derived_tasks`.
+
+```{r}
+hubExamples::forecast_outputs |>
+  dplyr::filter(output_type == "sample") |>
+  dplyr::mutate(output_type_id = as.numeric(output_type_id)) |> # make indices numeric for readability
+  hubEnsembles::linear_pool(
+    weights = NULL,
+    model_id = "linear-pool-joint",
+    task_id_cols = c("reference_date", "location", "horizon", "target", "target_end_date"),
+    compound_taskid_set = c("reference_date", "location", "target"),
+    derived_tasks = "target_end_date",
+    n_output_samples = 100
+  )
+```
+
+Derived task IDs are another subset of task ID variables whose values are "derived" from a combination of the values from other task ID variables, which may or may not be part of the compound task ID set. In the example, the derived task ID `"target_end_date"` is not part of the compound task ID set because `"reference_date"` and `"horizon"` are not both part of the compound task ID set.  
```

Comment:
These sentences are slightly redundant with text in the last sentence just before the code block, and expands on ideas that you started to introduce there.  How about removing it from here and combining with the previous paragraph?

Reply:
I take this suggestion back.